### PR TITLE
Update Description

### PR DESCRIPTION
**Pull request by smed79**

### CODE REVIEW

**Comment by LanikSJ** [comment]
As a general comment Travis isn't passing since GitHub can't figure out which repo/branch you used for this PR.  Next time fork this repo and create a PR from your fork.

File: filters/adback-domains.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [Adblock Plus 2.0]
-! Checksum: 00jxqNJaA0ZxDpVPpetCyA
+! Checksum: jRGiviJj69YwEv3xFITU7g
 ! Title: AdBack Domains Filter List
-! Last modified: 25 Jan 2019 10:33 UTC-7
+! Last modified: 2019-01-30 20:18 UTC
```
[the remark this answers is not in the document]
Comment (smed79) [reply filters/adback-domains.txt]: > I'd like to keep the dates consistent

Sorry for the unwanted change that i did not notice (i have used a different script for adding the Checksum)

> if you could change it to match the rest of the files

For unknown reason github does not let me edit the file and show me the message below:
_You must be signed in and have push access to the master branch to make changes._

[the remark this answers is not in the document]
Comment (LanikSJ) [reply filters/adback-domains.txt]: Might have to do with the branch being unknown.  I'll fix it no problem. 👍

[the remark this answers is not in the document]
Comment (smed79) [reply filters/adback-domains.txt]: Thank you.

**Pull request merged**